Source/Utility/Utility.cs

```diff
@@ -121,13 +121,12 @@ public static class Utility {
         }
 
         public static bool IsValidLoadoutHolder(this Pawn pawn) {
-            return pawn.RaceProps.Humanlike
-                   && !pawn.RaceProps.Animal
-                   && pawn.IsColonist
-                   && !pawn.Dead
-                   && !pawn.IsQuestLodger()
-                   && !(pawn.apparel?.AnyApparelLocked ?? true)
-                   && !pawn.RaceProps.packAnimal; // for a mod which has animals which somehow pass all of the above criteria...
```

Comment:
I don't understand the need for the packAnimal check, this is what causes many of the PM morphs to not be able considered valid because they inherit many of the information from the associated animal.
Using packAnimal to exclude animals would only cover a subset of animals too?